src/ripple/app/ledger/impl/LocalTxs.cpp

```diff
@@ -156,13 +156,12 @@ class LocalTxsImp : public LocalTxs
                 return true;
 
             AccountID const acctID = txn.getAccount();
-            auto const sleAcct = view.read(keylet::account(acctID));
+            auto const acctRoot = view.read(keylet::account(acctID));
```

Comment:
As far as the organization of the code is concerned, isn't it cleaner to use a constructor from the `AccountRootKeylet` class instead of the existing `keylet` namespace? I feel all the `keylet::<<lt_construction_functions>>` could be moved to their respective Strongly typed classes

Reply:
yes, thank you